### PR TITLE
Enable logging. 

### PR DESCRIPTION
**Pull request by wujingyue**
See code comments for the motivation.

### CODE REVIEW

File: tests/cpp/multidevice.cpp [hunk 1]
```diff
@@ -21,6 +21,11 @@
 namespace nvfuser {
 
 MultiDeviceTest::MultiDeviceTest() {
+  // Enable logging. Print debug messagees when knobs like
```
Comment (naoyam) [inline tests/cpp/multidevice.cpp:24]: Should this be enabled for all tests?

Comment (wujingyue) [reply tests/cpp/multidevice.cpp:24]: Call c10::initLogging in NVFuserTest and leave c10d::setDebugLevelFromEnvironment here? (That would make sense to me.)

Comment (naoyam) [reply tests/cpp/multidevice.cpp:24]: I was thinking moving both to NVFuserTest. Is there any reason that you may want to do `setDebugLevelFromEnvironment` only with the multi device tests?

Comment (naoyam) [reply tests/cpp/multidevice.cpp]: I see. I missed `d` of `c10d`. Makes sense.

**Review by naoyam — APPROVED** [review]
LGTM

**Comment by wujingyue** [comment]
!build

**Comment by wujingyue** [comment]
!build

**Comment by wujingyue** [comment]
!build

**Pull request merged**